src/docs/get-started/codelab.md

```diff
@@ -427,104 +437,96 @@ lazily, on demand.
+        that the divider may be difficult to see on smaller devices.
+     2. Add a one-pixel-high divider widget before each row in the `ListView`.
+     3. The expression `i ~/ 2` divides `i` by 2 and returns an integer result.
+        For example: 1, 2, 3, 4, 5 becomes 0, 1, 1, 2, 2. This calculates the
+        actual number of word pairings in the `ListView`, minus the divider
+        widgets.
+     4. If you've reached the end of the available word pairings, then generate
+        10 more and add them to the suggestions list.
+
     The `_buildSuggestions()` function calls `_buildRow()` once per
     word pair. This function displays each new pair in a `ListTile`,
     which allows you to make the rows more attractive in the next step.
 
  3. Add a `_buildRow()` function to `RandomWordsState`:
 
     <!-- skip -->
+    <?code-excerpt "4-infinite-list/lib/main.dart (_buildRow)" indent-by="2"?>
     {% prettify dart %}
-      class RandomWordsState extends State<RandomWords> {
-        ...
-
-        [[highlight]]Widget _buildRow(WordPair pair) {[[/highlight]]
-          [[highlight]]return ListTile([[/highlight]]
-            [[highlight]]title: Text([[/highlight]]
-              [[highlight]]pair.asPascalCase,[[/highlight]]
-              [[highlight]]style: _biggerFont,[[/highlight]]
-            [[highlight]]),[[/highlight]]
-          [[highlight]]);[[/highlight]]
-        [[highlight]]}[[/highlight]]
+      Widget _buildRow(WordPair pair) {
+        return ListTile(
+          title: Text(
+            pair.asPascalCase,
+            style: _biggerFont,
+          ),
+        );
       }
     {% endprettify %}
 
- 4. Update the `build` method for RandomWordsState to use
+ 4. Update the `build()` method for `RandomWordsState` to use
     `_buildSuggestions()`, rather than directly calling the word
     generation library.
     ([Scaffold](https://docs.flutter.io/flutter/material/Scaffold-class.html)
     implements the basic Material Design visual layout.)
 
+    Replace the method with the following code:
+
     <!-- skip -->
+    <?code-excerpt "4-infinite-list/lib/main.dart (RWS-build)" indent-by="2"?>
     {% prettify dart %}
-      class RandomWordsState extends State<RandomWords> {
-        ...
-        @override
-        Widget build(BuildContext context) {
-          [[strike]]final wordPair = WordPair.random();[[/strike]] // Delete these two lines.
-          [[strike]]return Text(wordPair.asPascalCase);[[/strike]]
-          [[highlight]]return Scaffold ([[/highlight]]
-            [[highlight]]appBar: AppBar([[/highlight]]
-              [[highlight]]title: Text('Startup Name Generator'),[[/highlight]]
-            [[highlight]]),[[/highlight]]
-            [[highlight]]body: _buildSuggestions(),[[/highlight]]
-          [[highlight]]);[[/highlight]]
-        }
-        ...
+      @override
+      Widget build(BuildContext context) {
+        return Scaffold(
```

Comment:
Perhaps highlight the contents? It seems OK to me to not highlight completely new methods, but when you're replacing the innards of a method, highlights seem appropriate.

Reply:
Yes, this will be changed to a pretty diff soon.

Reply:
I've added the highlighting because the simple diff plugin can't handle this case.